src/pyodide-py/pyodide/_core.py

```diff
@@ -16,5 +22,23 @@ class JsProxy:  # type: ignore
 
         # Defined in jsproxy.c
 
+        # Defined in jsproxy.c
+
+    def create_once_callable(obj: Callable) -> JsProxy:
+        """Wrap a Python callable in a Javascript function that can be called
+        once. After being called the proxy will decrement the reference count
```

Comment:
FYI, we use the numpydoc convention and [docstring need a 1 line summary](https://numpydoc.readthedocs.io/en/latest/format.html#sections), followed by an empty line, and then the rest of the docstring.

Though it seems to be rendering fine in [sphinx table summary](https://24620-122663163-gh.circle-artifacts.com/0/home/circleci/repo/docs/_build/html/usage/api-reference.html) so it's probably for to keep it as is.